_layouts/post.liquid

```diff
@@ -12,6 +12,10 @@
       <div class="col-md-8">
         <main class="blog">
           <article>
+            <h1>{{ title }}</h1>
+            <em>{{ blogDate }}</em>
```

Comment:
```suggestion
            <em>{{ page.date.toDateString() }}</em>
```

Reply:
I updated to the following code in the layout: 

`<em>{{ page.date.toDateString() }}</em>`

When I run the code the following is displayed, the date is not displayed:

<img width="794" alt="Screen Shot 2021-09-20 at 9 02 30 PM" src="https://user-images.githubusercontent.com/47087608/134097089-265afada-f771-4b00-899c-af6baef8c15a.png">

Reply:
I'm now sure if `liquidjs` allows function calls here. For date formatting, we're using `luxon` as a custom filter.

If we want the same date format as in the list of recent posts (which is on the same page, on the right side), it would be:

```
{{ page.date | dateFormat: "d LLLL yyyy" }}
```

An example with a formatted date is in the same file:

https://github.com/eslint/website/blob/10ecb8240da6507794d08d9937b9a0827ba01180/_layouts/post.liquid#L33

If we want another format, here's [table of tokens](https://moment.github.io/luxon/#/formatting?id=table-of-tokens).

Reply:
@thomasgilmore95 that’s weird. I took that from an Eleventy example. 

Using Luxon is a better idea anyway  👍

Reply:
Thank you. I will change to use Luxon.